app/views/govuk_publishing_components/components/docs/list.yml

```diff
@@ -0,0 +1,59 @@
+name: List
+description: A list, unordered or ordered.
+body: |
+  This is an ordered or unordered list, with or without visible bullets / numbers.
+accessibility_criteria: |
+  The list must:
+
+  - inform the user how many items are in the list
+  - convey the content structure
+  - indicate the current page when contents span different pages, and not link to itself
+
+  The contents list may:
+
+  - include an aria-label to contextualise the list if helpful
+shared_accessibility_criteria:
+  - link
+  - contents_list
```

Comment:
I don't think there's a shared accessibility criteria for `contents_list`? This doesn't seem to output anything.

Reply:
Boo. Removed.